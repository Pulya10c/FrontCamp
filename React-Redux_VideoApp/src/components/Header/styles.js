import styled from 'styled-components';
import blurImage from './blurImage.jpg';

export const Header = styled.header`
  display: grid;
  grid-gap: 10px;
  justify-content: center;
  align-content: center;

  padding: 10px;
  min-height: 200px;
  background-image: url(${blurImage});
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
`;
