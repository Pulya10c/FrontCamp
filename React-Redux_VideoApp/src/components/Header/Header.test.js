import React from 'react';
import { shallow } from 'enzyme';
import Header from './';

describe('Component Header', () => {
  it('should render correctly', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
