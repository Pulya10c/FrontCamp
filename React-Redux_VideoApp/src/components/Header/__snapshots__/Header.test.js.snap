// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Header should render correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  display: grid;
  grid-gap: 10px;
  justify-content: center;
  align-content: center;

  padding: 10px;
  min-height: 200px;
  background-image: url(",
          "blurImage.jpg",
          ");
  background-repeat: no-repeat;
  background-position: center;
  background-size: cover;
",
        ],
      },
      "displayName": "styled.header",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "header",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
